.travis.yml

```diff
@@ -2,12 +2,14 @@ language: ruby
 sudo: false
 before_install:
   - gem install bundler -v 1.10
+  - gem update --system --no-doc
```

Comment:
Note that is unfortunately due to the `rainbow` gem and some incompatibilities with rubygems, requiring rubygems `2.6.9` or higher.

That issue is tracked here - https://github.com/sickill/rainbow/issues/44

Might be worth seeing what is pulling in rainbow and if it can be removed in any way, as an alternate option.